.github/scripts/.helm-tests/Openshift/values.yaml

```diff
@@ -0,0 +1,18 @@
+global:
+  isOpenshift: true
+certManager:
+  image:
+    tag: v0.0.0
+lifecycleOperator:
+  enabled: true
+  lifecycleOperator:
+    image:
+      tag: v0.0.0
+      imagePullPolicy: Never
+  scheduler:
+    image:
+      tag: v0.0.0
+metricsOperator:
+  enabled: true
+  image:
+    tag: v0.0.0
```

Comment:
you could remove all those values since they are not relevant for this test

Reply:
I included all those those tags=v0.0.0 to minimize the brittleness of the test. Yes, I copied them from another test. :-)